source/UI.cpp

```diff
@@ -240,7 +240,7 @@ bool UI::IsDone() const
 
 
 
-// Check if it is time to quit.
+// Check if there are no panels left.
```

Comment:
I don't feel this is necessary unless it is called in circumstances other than checking if we entered the quit state from UI interactions.

Reply:
The auto-test code from #4370 also checks if the menu-stack is empty to determine if the game is running or if we are in a menu.
I extended the text a little bit to make the usage of this code more clear.